File: practicas/3/SeguraCedeñoLuisaMaria/saludo.c
#include <stdio.h>
#include <string.h>

int main(int argc, char *argv[]) {
  if (argc > 1) {
    printf("Hola, %s!\n", argv[1]);
  } else {
    printf("Hola, Mundo!\n");
  }
  return 0;
}

File: practicas/3/RodriguezAlfonso/saludo.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
)

func main() {
	fmt.Println("Introduzca su nombre:")
	reader := bufio.NewReader(os.Stdin)
	line, err := reader.ReadString('\n')
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("%s mundo! Digo, Hola %s", line, line)
}
